tools/wasm_demo/CMakeLists.txt

```diff
@@ -18,27 +18,30 @@ set(JXL_DECODER_SYMBOLS
 )
 list(JOIN JXL_DECODER_SYMBOLS ", " JXL_DECODER_EXPORTS)
 
-set_target_properties(jxl_decoder PROPERTIES LINK_FLAGS "\
-  -O3 \
-  --closure 1 \
-  -s TOTAL_MEMORY=75mb \
+# TODO(eustas): How to set TOTAL_MEMORY on module startup?
+set(JXL_DECODER_BASE_LINK_FLAGS "\
   -s USE_LIBPNG=1 \
+  -s TOTAL_MEMORY=75mb \
```

Comment:
FWIW, this setting was renamed to [INITIAL_MEMORY](https://emsettings.surma.technology/#INITIAL_MEMORY) in Emscripten v1.39.9.  The old names are still supported as aliases.

Reply:
Thanks! Fixed.